R/gh_token.R

```diff
@@ -164,21 +180,162 @@ gh_auth <- function(token) {
 #' # A custom one
 #' slugify_url("https://github.acme.com")
 
-slugify_url <- function(url) {
+slugify_url <- function(url) {           # https://jane@github.uni.edu/api/v3
+  url <- get_baseurl(url)                # https://jane@github.uni.edu
+  url <- normalize_host(url)
+  x2 <- sub("^.*://([^/]*@)?", "", url)  # github.uni.edu
+  x3 <- gsub("[.]+", "_", x2)            # github_uni_edu
```

Comment:
@gaborcsardi This regex changed (lost the `/`) because we've already truncated to the base url and should no longer have any `/`.